eslint.config.mjs

```diff
@@ -36,10 +36,6 @@ export const baseConfig = [
         },
       ],
 
-      indent: ['error', 2],
```

Comment:
FYI, I had an old PR removing some of these, but there is a gap because Prettier isn't enabled on JS files, so these were kept to backfill the gap while things were converted to TS

Reply:
oh that's good to know, thank you.